script/strip-binaries.py

```diff
@@ -29,15 +29,49 @@ def strip_binary(binary_path, target_cpu):
     strip = 'mips64el-redhat-linux-strip'
   else:
     strip = 'strip'
+  extract_symbols(binary_path, target_cpu, compress)
```

Comment:
This should be done outside of the `strip_binary` function.
Please move it to a separate script and call it in `.circleci/config.yml` before calling `strip-binaries.py`.

Reply:
I cannot split because of the 2 objcopy execution that needs to surrounds the strip command, see my other comment

Reply:
Done I created a separate script, thx for the suggestion!